ArduPlane/quadplane.cpp

```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
-    // @Description: This provides a set of additional control options for quadplanes. LevelTransition means that the wings should be held level to within LEVEL_ROLL_LIMIT degrees during transition to fixed wing flight, and the vehicle will not use the vertical lift motors to climb during the transition. If AllowFWTakeoff bit is not set then fixed wing takeoff on quadplanes will instead perform a VTOL takeoff. If AllowFWLand bit is not set then fixed wing land on quadplanes will instead perform a VTOL land. If respect takeoff frame is not set the vehicle will interpret all takeoff waypoints as an altitude above the current position. When Use QRTL is set it will replace QLAND with QRTL for failsafe actions when in VTOL modes. When AIRMODE is set AirMode is automatically enabled if arming by RC channel.
-    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect takeoff frame types,4:Use a fixed wing approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use idle governor in MANUAL,7:QAssist force enabled,8:Tailsitter QAssist motors only,9:enable AirMode if arming by aux switch,10:Enable motor tilt when disarmed,11:Delay spoolup for 2 seconds after arming
+    // @Description: This provides a set of additional control options for quadplanes. LevelTransition means that the wings should be held level to within LEVEL_ROLL_LIMIT degrees during transition to fixed wing flight, and the vehicle will not use the vertical lift motors to climb during the transition. If AllowFWTakeoff bit is not set then fixed wing takeoff on quadplanes will instead perform a VTOL takeoff. If AllowFWLand bit is not set then fixed wing land on quadplanes will instead perform a VTOL land. If Respect Takeoff Frame is not set the vehicle will interpret all takeoff waypoints as an altitude above the current position rather than above HOME. If Use Fixed Wing Approach is set, VTOL landing will fly and loiter to alt before VTOL landing. If Use QRTL is set it will replace QLAND with QRTL for failsafe actions when in VTOL modes. If Use Idle Governor is set, idle governor will be used to set thr_min in MANUAL mode for VTOL motors. If QAssistT Force is set, QAssist is active in all VTOL modes while armed. If Tailsitter QAssist is set, flying surfaces are not used while in QAssist. If Enable AIRMODE is set AirMode is automatically enabled after arming by an RC channel switch. If Enable Motor Tilt is set, when disarmed, tilt servos move with rudder input. If Delay spoolup is set, VTOL motors spin up is delayed after arming.
+    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect Takeoff Frame,4:Use Fixed Wing Approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use Idle Governor in MANUAL,7:QAssist Force enabled,8:Tailsitter QAssist motors only,9:Enable AirMode if arming by aux switch,10:Enable Motor Tilt when disarmed,11:Delay Spoolup for 2 seconds after arming
```

Comment:
`Respect Takeoff Altitude Frame`, rest looks OK to me

Reply:
oops , missed fixing one..thanks!

Reply:
"Takeoff Frame" shouldn't be caps locked, it's not a proper noun/acronym. Same for "Fixed Wing Approach". Those were intentionally lower case.

Reply:
the mixture of short name and acronyms is in consistent....perhaps they should all be acronyms for easy referral in the description...makes it clearer in the Wiki...
LevelTransition,AllowFWTakeoff,RespectTKOFrame,UseFWApproach,UseQRTL,UseGovernor,ForceQAssist,MtrsOnlyQAssist,AirModeEn,DelaySpoolUp ? they appear as a table...

Reply:
I don't think acronyms would help honestly. There are already so many acronyms we all take for granted I think throwing more (very non standard) acronyms at people for this aren't going to help. In general I'd say anything that is short like `RespectTKOFrame` is actually user hostile. It's both not obvious to the reader what TKO is, and to make it worse it actively makes it harder for people who don't have english as their first language, as that type of thing is nearly impossible to get through a translation tool, or get a handle on with a dictionary, whereas spelling it out as `Respect the takeoff altitude frame` is quite clear in English, as well as having a chance to be used in other languages.

Reply:
okay, then going the other way for consistency...no need for a long description if its in the param bitmask....checked MP and wiki and they would display correctly...